packages/card-react/documentation/TaskCardExample.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { ExampleComponentProps } from "../../../doc-utils";
+
+import { Checkbox } from "../../checkbox-react/src";
+import { Select } from "../../select-react/src";
+import { FieldGroup } from "../../field-group-react/src";
+import { useAnimatedHeight } from "../../react-hooks";
+import { TaskCard } from "../src/TaskCard";
+import { mixedPadding } from "./cardExampleProps";
+
+import "./task-card-example.scss";
+
+export const TaskCardExample: React.FC<ExampleComponentProps> = ({ choiceValues }) => {
+    const [selected, setSelected] = useState(false);
+    const [contentRef] = useAnimatedHeight<HTMLDivElement>(selected);
+
+    const paddingChoice = (choiceValues?.["Padding"] as "m" | "l" | "xl" | "blandet") || "l";
+    const padding = paddingChoice === "blandet" ? mixedPadding : paddingChoice;
+
+    return (
+        <TaskCard bgColor={selected ? "hvit" : "dis"} shadow={selected} padding={padding}>
+            <Checkbox name="veterinar" value="veterinar" onChange={() => setSelected((sel) => !sel)}>
+                Veterinærutgifter
+            </Checkbox>
+            <p className="jkl-body jkl-spacing-s--top">
+                Dekker veterinærutgifter og medisiner på resept, som ofte kan bli dyrt.
+            </p>
+            <div className="jkl-task-card-example__content" hidden={!selected} ref={contentRef}>
```

Comment:
Kom på her jeg ser igjennom kodeeksempelet i portalen; her kan det være fint å følge [tilgjengelighets-docs for ExpandButton](https://jokul.fremtind.no/komponenter/expandbutton#tilgjengelighet). La Checkbox fylle rollen til ExpandButton, og denne diven være innholdet som styres.

Reply:
Ja, det er et godt poeng! Skal slenge på riktige roller og props :)